pyproject.toml

```diff
@@ -86,7 +86,6 @@ max-complexity = 17  # default: 10
 "graphs/minimum_spanning_tree_prims.py" = ["SIM114"]
 "hashes/enigma_machine.py" = ["BLE001"]
 "machine_learning/decision_tree.py" = ["SIM114"]
-"machine_learning/linear_discriminant_analysis.py" = ["ARG005"]
 "machine_learning/sequential_minimum_optimization.py" = ["SIM115"]
```

Comment:
Guess we shouldn't as to change the code !!

Reply:
Why do you "guess" that?

Reply:
Is that simply great to remove a line?

Reply:
Is that great while removing a line, while making it a fix !!